magento/pwa-studio
# [PWA-618] Update Venia's header to match the styleguide's app bar

## Description

Update app header to reflect the new design

## Related Issue





Closes https://jira.corp.magento.com/browse/PWA-618

## Acceptance 



### Verification Stakeholders
@jimbo
@dhaecker
### Specification



### Verification Steps


* `Header` styles to match the new design
* Reference the `app bar` component from `venia-styleguide` for sizing and layout
* New Venia logo. Logo should display right of mega menu
    * Check if overwriting logo will result in regression issues in other areas
* Retain breakpoints and check if components break. Breaking components should switch to desktop.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
